precompiles/ArbSys.go

```diff
@@ -174,7 +175,9 @@ func (con *ArbSys) SendTxToL1(c ctx, evm mech, value huge, destination addr, cal
 	}
 
 	// burn the callvalue, which was previously deposited to this precompile's account
-	evm.StateDB.SubBalance(con.Address, value)
+	if err := util.BurnBalance(&con.Address, value, evm, util.TracingDuringEVM, "l1Send"); err != nil {
```

Comment:
How about "withdraw"? "l1Send" could imply either direction.

Reply:
Changed to `withdraw`, though since this happens during EVM execution it won't show up either way